### DISCUSSION TITLE
Interactive Plotting

### DISCUSSION THREAD

**Discussion by scottstephens** [dotnet/interactive#3176] [Q&A]
I've been using VSCode Polyglot Notebooks and XPlot.Plotly for plotting. 

Are there any good ways to drive a plot using sliders, or spinners, or other data entry widgets?

**Comment by Gryhyphen** [comment]
Am late to the party, but I've had success building interactive charts using echarts in javascript + html.

Was doing it purely in f# using plotly.net but problem is that output doesn't support building custom ui / dealing with callbacks. Better to just shove it into javascript land and render it using the popular js charting libraries available. Plotly.js api, is, imo, awful, which is why I used echarts.
